### PR TITLE
Mentor show

### PR DESCRIPTION
**Pull request by jbkimble**
@meganft @bfpepper @vidoseaver Hello all, the mentor show page is currently functional.  We'll have to change how we access the mentor for the show page to an ajax call to the census API when we have access to census.  Otherwise all should be good!

### CODE REVIEW

File: app/assets/javascripts/components/show/_show.js.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+var Show = React.createClass({
+  componentWillMount() {
+    console.log(this.props.mentor[0].name)
```
Comment (neight-allen) [inline app/assets/javascripts/components/show/_show.js.jsx]: Try to get into the habit of removing `console.log` before opening a PR

File: app/assets/javascripts/components/show/_show.js.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          </span>
+          <h3 className="col s12">{this.props.mentor[0].name}</h3>
+          <p>Last Active: {this.props.mentor[0].last_active} </p>
+          <p>Location: {this.props.mentor[0].location} </p>
+          <p>Company: {this.props.mentor[0].position} @ {this.props.mentor[0].company} </p>
+          <p>Email: {this.props.mentor[0].email} </p>
+          <p>Slack: {this.props.mentor[0].slack_username} </p>
+          <p>Expertise: {this.props.mentor[0].expertise.replace(/"/g, "").replace("[", "").replace("]", "")}</p>
+          <p>Position: {this.props.mentor[0].position} </p>
+          <p>Bio: {this.props.mentor[0].bio} </p>
```
Comment (neight-allen) [inline app/assets/javascripts/components/show/_show.js.jsx]: I'm not sure if this is normal style, but I'd prefer we start adding something like `let mentor = this.props.mentor[0]` at the top of the render function, so you can just write `mentor.avatar` in the tags below.

Also, why is the mentor property being passed as an array for a show page?

File: app/views/mentors/show.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<%= react_component 'Show', {mentor: @mentor}, :div %>
```
Comment (neight-allen) [inline app/views/mentors/show.html.erb]: I'm leaning towards making the navbar an ERB thing. It doesn't have to change for me to accept this PR, but I think especially with oauth being on the server side, it's going to make more sense to render the navbar in the layout. We can discuss it more in person.

**Comment by jbkimble** [comment]
@neight-allen made the changes you mentioned.  The reason that the object is being passed through as an array is because react-rails requires objects being passed through the controller to be inside an array.  I like the idea of having the nav bar live on the backend but we'll discuss in person.  Thanks!

**Comment by neight-allen** [comment]
I'm still researching why the test passes without there being any Javscript driver in your rspec, but as soon as you resolve the merge conflict, this is ready to go.

**Comment by jbkimble** [comment]
@neight-allen resolved all merge conflicts!

**Pull request merged**